src/aat/resources/features/F-037/F-037_Test_Data_Base.td.json

```diff
@@ -0,0 +1,73 @@
+{
+	"_guid_": "F-037_Test_Data_Base",
+	"productName": "CCD Data Store",
+	"operationName": "Create event (V2)",
+	"method": "POST",
+	"uri": "/cases/{cid}/events",
+
+	"user": {
+		"_extends_": "Common_User_For_Request"
+	},
+	"request": {
+    "headers": {
+      "_extends_": "Common_Request_Headers",
+      "Content-Type": "application/vnd.uk.gov.hmcts.ccd-data-store-api.create-event.v2+json",
+      "experimental": true
+    },
+    "body": {
```

Comment:
We can leverage the JURISDICTION1 case type which hardly contains 6 fields, thus reducing the size of the file and utilizing the dedicated jurisdictions for the FTA task.

Reply:
Yes it can save few lines but I am not planning to change the jurisdiction for now.